doc/operation_guides/manual/backup-guide.md

```diff
@@ -0,0 +1,157 @@
+# Backup Guide
+
+We advice to backup your OpenProject installation regularly — especially
+before upgrading to a newer version.
+
+## Backup the Database
+
+###OpenProject Version 3.0.15 and newer
+
+Execute the following command in a shell in the directory where
+OpenProject is installed:
+
+````bash
+RAILS_ENV=production bundle exec rake backup:database:create
+```
+
+The command will create dump of your database which can be found at
+
+`OPENPROJECT_DIRECTORY/backup/openproject-production-db-<DATE>.sql` (for
+MySQL) or
+`OPENPROJECT_DIRECTORY/backup/openproject-production-db-<DATE>.backup`
+(for PostgreSQL).
+
+Optionally, you can specify the path of the backup file. Therefore
+you have to replace the `/path/to/file.backup` with the path of your
+choice
+
+```bash
+RAILS_ENV=production bundle exec rake backup:database:create[/path/to/backup/file.backup]
+```
+*Note:* You can restore any database backup with the following
+command. Be aware that you have to replace the
+`/path/to/backup/file.backup` path with your actual backup path.
+
+```bash
+RAILS_ENV=production bundle exec rake backup:database:restore[/path/to/backup/file.backup]
+```
+
+If your database dump is from an old version of OpenProject, also run
```

Comment:
"also run" does not indicate the timing very well.

I assume that you should restore first and migrate afterwards. Better make that clear, e.g.

> If your database dump is from an old version of OpenProject, also run the following command after restoring: